foreman/foreman.spec

```diff
@@ -158,6 +160,8 @@ BuildRequires: %{?scl_prefix}rubygem(deep_cloneable) >= 2.2.2
 BuildRequires: %{?scl_prefix}rubygem(deep_cloneable) < 3.0
 BuildRequires: %{?scl_prefix}rubygem(validates_lengths_from_database) >= 0.5
 BuildRequires: %{?scl_prefix}rubygem(validates_lengths_from_database) < 1.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.5
```

Comment:
I don't think we need the tasks in the build phase, the only thing I can think of si the class loading of dynflow actions, which we could workaround if needed. The other solution might be removing foreman from foreman-tasks dependencies, as the foremen-tasks usually get installed only as dependency of something else, so the `foreman` it's guaranted to be around.

I'm trying now without the build dependency specified to see if there are any issues there.